components/support/ktx/src/test/java/mozilla/components/support/ktx/android/graphics/BitmapKtTest.kt

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.ktx.android.graphics
+
+import android.graphics.Bitmap
+import android.graphics.Color
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotSame
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class BitmapKtTest {
+
+    @Ignore // TODO: convert to integration test. Robolectric's shadows are incomplete and cause this to fail.
```

Comment:
That's something we could do in a follow-up, right?

Reply:
Yeah – it doesn't look like components supports integration tests right now so I didn't try to set it up. Filed https://github.com/mozilla-mobile/android-components/issues/1637